### PR TITLE
chore(vscode): update vscode setting file

### PR DESCRIPTION
**Pull request by sokol147**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/nebular/blob/master/CONTRIBUTING.md) guide.
 - [x] I read and followed the [New Feature Checklist](https://github.com/akveo/nebular/blob/master/DEV_DOCS.md#new-feature-checklist) guide.
 
 #### Short description of what this resolves:
 Add stylelint settings to vscode/setting.json

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
   "typescript.tsdk": "./node_modules/typescript/lib",
```
Comment (yggg) [inline .vscode/settings.json:2]: Do we still need this line?

**Review by yggg — COMMENTED** [review]
Could you also update `"**/.lib": true` to `"**/dist": true`, please?. `.lib` directory was removed.

File: .vscode/settings.json (hunk 1 above)
Comment (sokol147) [reply .vscode/settings.json]: deleted

File: .vscode/settings.json [hunk 2]
```diff
@@ -15,6 +16,6 @@
     "**/*.js.map": true,
 
     // exclude @nebular build folders
-    "**/.lib": true
+    "**/dist": true
```
Comment (yggg) [inline .vscode/settings.json:19]: Shouldn't we also exclude `.angular` and `packages-smoke/.angular` directories?

**Review by yggg — APPROVED** [review]

**Pull request merged**